openvmm/hvlite_core/src/worker/vm_loaders/igvm.rs

```diff
@@ -496,11 +498,14 @@ fn build_device_tree(
             .end_node()?;
     }
 
-    // Indicate that NVMe keep-alive feature is supported by this VMM.
-    openhcl = openhcl
-        .start_node("keep-alive")?
-        .add_str(p_vf_keep_alive_devs, "nvme")?
-        .end_node()?;
+    // Default to enabling NVMe keep-alive unless explicitly disabled.
+    if device_tree_overrides.is_none() || device_tree_overrides.unwrap().nvme_keepalive_enable {
```

Comment:
jfyi, you could use a construct like this. What you have is fine. But sharing the latest idiomatic trick I learned.

```
match device_tree_overrides {
    None || Some( Overrides { nvme_keepalive_enable: false } )
```